index.html

```diff
@@ -601,6 +601,402 @@ <h6 class="float-left">START YOUR DAY WITH OUR TOP 5 ARTICLES</h6>
+                </div>            
+                <div class="container-fluid px-3 pb-3 col-md-4">                
+                    <div class="py-3 h-auto">
+                        <a href="#"><img class="float-start w-50 my-auto pe-2" src="./img/us-pic-thumb.webp" alt="homeless"></a>                
+                        <strong><a href="#" class="text-decoration-none text-dark small">Trump Pardons Anthony Levandowski, Engineer Who Stole Google Trade Secrets</a></strong>
+                        <div class="clearfix"></div>
+                    </div>
+                    <div class="py-3 border-top">
+                        <a href="#"><img class="float-start w-50 my-auto pe-2" src="./img/us-pic-thumb.webp" alt="homeless"></a>                
+                        <strong><a href="#" class="text-decoration-none text-dark small">Trump Pardons Anthony Levandowski, Engineer Who Stole Google Trade Secrets</a></strong>
+                        <div class="clearfix"></div>
+                    </div>
+                    <div class="py-3 border-top">
+                        <a href="#"><img class="float-start w-50 my-auto pe-2" src="./img/us-pic-thumb.webp" alt="homeless"></a>                
+                        <strong><a href="#" class="text-decoration-none text-dark small">Trump Pardons Anthony Levandowski, Engineer Who Stole Google Trade Secrets</a></strong>
+                        <div class="clearfix"></div>
+                    </div>
+                </div>
+            </div>
+            
+        </section>
+        <section id="tech-science">
+            <div class="caption d-flex align-items-center gap-2 mt-3 justify-content-start text-center pt-3 mx-3 border-top">                
+                <strong class="fs-4 text-danger">TECH & SCIENCE</strong>            
+            </div>
+            <div class="d-flex flex-column flex-md-row">
+                <div class="container-fluid px-3 py-3 col-md-4">                    
+                    <a href="#"><img class="w-100" src="./img/us-pic.webp" alt="crazy dude"></a>                    
+                    <strong><a href="#" class="text-decoration-none text-muted">Why Netflix No Longer Offers a Free Trial</a></strong>
+                </div>
+                <div class="container-fluid px-3 py-3 col-md-4">                    
+                    <a href="#"><img class="w-100" src="./img/us-pic.webp" alt="crazy dude"></a>                    
+                    <strong><a href="#" class="text-decoration-none text-muted">Why Netflix No Longer Offers a Free Trial</a></strong>
+                </div>            
+                <div class="container-fluid px-3 pb-3 col-md-4">                
+                    <div class="py-3 h-auto">
+                        <a href="#"><img class="float-start w-50 my-auto pe-2" src="./img/us-pic-thumb.webp" alt="homeless"></a>                
```

Comment:
These images should be smaller and correctly aligned as in the original website.